static/app/views/insights/settings.ts

```diff
@@ -196,9 +198,22 @@ export const MODULE_FEATURE_MAP: Record<ModuleName, string[]> = {
 };
 
 /**
- * Modules that will not display tabs when the feature is not enabled
+ * Features that control the visibility of modules
  */
-export const MODULE_HIDDEN_WHEN_FEAUTRE_DISABLED: ModuleName[] = [
-  ModuleName.CRONS,
-  ModuleName.UPTIME,
-];
+export const MODULE_FEATURE_HIDDEN_MAP: Record<ModuleName, string[]> = {
```

Comment:
This names confused me for a moment, I thought these flags might be negations, as in if these flags are present the feature is hidden!

Maybe just a me problem, but I think `MODULE_VISIBLE_FEATURE_MAP` is better